pytest passes locally but fails on github: hypothesis.errors.FailedHealthCheck

looper is failing on dev branch with a new error during pytest on github. However, the tests are all passing locally.
Appears to be related to hypothesis module.

```
=================================== FAILURES ===================================
______ NaturalRangeFromStringTests.test_just_delimiter__does_not_parse[-] ______

self = <tests.test_natural_range.NaturalRangeFromStringTests object at 0x7f4ea9d2b850>
legit_delim = '-'

    @given(upper_bound=st.integers())
>   def test_just_delimiter__does_not_parse(self, legit_delim, upper_bound):
E   hypothesis.errors.FailedHealthCheck: The method NaturalRangeFromStringTests.test_just_delimiter__does_not_parse was called from multiple different executors. This may lead to flaky tests and nonreproducible errors when replaying from database.
E   See https://hypothesis.readthedocs.io/en/latest/healthchecks.html for more information about this. If you want to disable just this health check, add HealthCheck.differing_executors to the suppress_health_check settings for this test.

tests/test_natural_range.py:65: FailedHealthCheck
=========================== short test summary info ============================
FAILED tests/test_natural_range.py::NaturalRangeFromStringTests::test_just_delimiter__does_not_parse[-] - hypothesis.errors.FailedHealthCheck: The method NaturalRangeFromStringTests.test_just_delimiter__does_not_parse was called from multiple different executors. This may lead to flaky tests and nonreproducible errors when replaying from database.
See https://hypothesis.readthedocs.io/en/latest/healthchecks.html for more information about this. If you want to disable just this health check, add HealthCheck.differing_executors to the suppress_health_check settings for this test.
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
```

--- a/docs/README.md
+++ b/docs/README.md
@@ -1,6 +1,6 @@
 # <img src="img/looper_logo.svg" class="img-header"> pipeline submitting engine
 
-[![PEP compatible](http://pepkit.github.io/img/PEP-compatible-green.svg)](http://pepkit.github.io)
+[![PEP compatible](https://pepkit.github.io/img/PEP-compatible-green.svg)](http://pepkit.github.io)
 
 ## What is looper?
 

--- a/requirements/requirements-test.txt
+++ b/requirements/requirements-test.txt
@@ -1,4 +1,4 @@
-hypothesis
+hypothesis >= 6.84.3
 mock
 pytest
 pytest-cov

